includes/Modules/Subscribe_With_Google.php

```diff
@@ -0,0 +1,166 @@
+<?php
+/**
+ * Class Google\Site_Kit\Modules\Subscribe_With_Google
+ *
+ * @package   Google\Site_Kit
+ * @copyright 2021 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://sitekit.withgoogle.com
+ */
+
+namespace Google\Site_Kit\Modules;
+
+use Google\Site_Kit\Core\Assets\Asset;
+use Google\Site_Kit\Core\Assets\Script;
+use Google\Site_Kit\Core\Modules\Module;
+use Google\Site_Kit\Core\Modules\Module_Settings;
+use Google\Site_Kit\Core\Modules\Module_With_Assets;
+use Google\Site_Kit\Core\Modules\Module_With_Assets_Trait;
+use Google\Site_Kit\Core\Modules\Module_With_Settings;
+use Google\Site_Kit\Core\Modules\Module_With_Settings_Trait;
+use Google\Site_Kit\Core\Util\Method_Proxy_Trait;
+use Google\Site_Kit\Modules\Subscribe_With_Google\EditPost;
+use Google\Site_Kit\Modules\Subscribe_With_Google\Header;
+use Google\Site_Kit\Modules\Subscribe_With_Google\InlineEditPost;
+use Google\Site_Kit\Modules\Subscribe_With_Google\ManagePosts;
+use Google\Site_Kit\Modules\Subscribe_With_Google\Settings;
+use Google\Site_Kit\Modules\Subscribe_With_Google\SinglePost;
+
+/**
+ * Class representing the Subscribe with Google module.
+ *
+ * @since 1.0.0
+ * @access private
+ * @ignore
+ */
+final class Subscribe_With_Google extends Module
+	implements Module_With_Assets, Module_With_Settings {
+	use Method_Proxy_Trait;
+	use Module_With_Assets_Trait;
+	use Module_With_Settings_Trait;
```

Comment:
Related to another comment below in your `Settings` class? Can you also implement `Module_With_Owner` here (as simple as using `Module_With_Owner_Trait`? This way this module also supports the concept of ownership, determining which user last configured settings for it.

Reply:
Added!